Libraries/LibWeb/CSS/Parser/ValueParsing.cpp

```diff
@@ -4052,6 +4052,11 @@ bool Parser::expand_variables(DOM::Element& element, Optional<PseudoElement> pse
 
     while (source.has_next_token()) {
         auto const& value = source.consume_a_token();
+        auto single_token = TokenStream<ComponentValue>::of_single_token(value);
+        if (auto builtin_value = parse_builtin_value(single_token)) {
+            if (builtin_value->is_css_wide_keyword())
+                continue;
+        }
```

Comment:
We can use `source` instead of creating a new TokenStream. And `is_css_wide_keyword()` will always return `true` for parsed builtin values. (They're the same thing, it's just a naming inconsistency.)

So this can be reduced to:

```suggestion
        if (auto builtin_value = parse_builtin_value(source))
            continue;
        auto const& value = source.consume_a_token();
```